data/sql/updates/pending_db_world/rev_1586465404537052600.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1586465404537052600');
+
+UPDATE `creature_addon` SET `auras`= '21862' WHERE `guid` = 201240;
```

Comment:
why `auras` as a string?

Reply:
All auras in creature_addon are strings

Reply:
Because you add auras like 'aura1 aura2' etc

Reply:
From wiki:
" To apply multiple auras, you can add more aura entries, separating each entry by a space. Remember that if a spell applies multiple auras."

Reply:
i don't see any string wtf?

Reply:
@Kitzunu fair enough, I forgot that `auras` is indeed a string and not a bitmask value

![image](https://user-images.githubusercontent.com/75517/79028336-e4f4a700-7b8f-11ea-956d-33f57fb570cf.png)

@BarbzYHOOL it's because I saw a number passed as a string:

![image](https://user-images.githubusercontent.com/75517/79028372-0786c000-7b90-11ea-9706-26ab0f009205.png)

but in this case it's correct